frontend/app/routes/companies.js

```diff
@@ -1,25 +1,5 @@
-import { inject as service } from '@ember/service';
 import Route from '@ember/routing/route';
 import AuthenticatedRouteMixin from 'ember-simple-auth/mixins/authenticated-route-mixin';
 
 export default Route.extend(AuthenticatedRouteMixin, {
```

Comment:
Das Laden der Firmen wird nun in der `companies.index` Route abgehandelt. Diese Änderung erübrigt das manuelle Aktualisieren der Firmenliste.